core/store/src/trie/mod.rs

```diff
@@ -1493,6 +1492,25 @@ impl TrieAccess for Trie {
     }
 }
 
+/// Counts trie nodes reads during tx/receipt execution for proper storage costs charging.
+#[derive(Debug, PartialEq)]
+pub struct TrieNodesCount {
```

Comment:
Why not to import it from near_vm_runner temporarily, if we depend on it anyway?

Reply:
So the most sensible option is to really have two types. One is an "output" of `near-store` and the other is used (but doesn’t need to be!) in the interface that `near-vm-runner` exposes via `trait Externals`.